topics/visualisation/metadata.yaml

```diff
@@ -0,0 +1,13 @@
+﻿---
+name: "visualisation"
+type: "use"
+title: "Visualisation"
+summary: "Training material for data visualisation in Galaxy"
+
+requirements:
+  - type: "internal"
+    topic_name: introduction
+
+maintainers:
+  - shiltemann
+  - erasche
```

Comment:
Needs to be updated maybe? ;)

Reply:
yeah coming in a separate PR since I have to edit like 20 different tutorials at once and didn't want the noise on this PR :/

Reply:
ok no problem! I meant to click on "Start a review", I'm currently reading it